ironfish/src/sdk.ts

```diff
@@ -67,6 +74,7 @@ export class IronfishSdk {
     pkg,
     configName,
     configOverrides,
+    internalConfigOverrides,
```

Comment:
this isn't config, its internal so it should be just `internalOverrides`

```suggestion
    internalOverrides,
```